src/views/search.hbs

```diff
@@ -1 +1,2 @@
+<link rel="stylesheet" type="text/css" href="/css/search.css" />
```

Comment:
Maybe all search related styling will live in `presentations.css` and this `search.css` file isn't necessary...?